apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.spec.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * @jest-environment node
+ */
+
+const dayjs = require('dayjs')
+
+const { setFakeClientMode } = require('@core/keystone/test.utils')
+const { sendSubmitMeterReadingsPushNotifications } = require('@condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications')
+const { Message: MessageApi } = require('@condo/domains/notification/utils/serverSchema')
+const {
+    METER_SUBMIT_READINGS_REMINDER_TYPE,
+    METER_VERIFICATION_DATE_EXPIRED_TYPE,
+} = require('@condo/domains/notification/constants/constants')
+const {
+    MeterReadingSource,
+    makeClientWithResidentAndMeter,
+    createTestMeterReading,
+} = require('@condo/domains/meter/utils/testSchema')
+const { CALL_METER_READING_SOURCE_ID } = require('@condo/domains/meter/constants/constants')
+
+const index = require('@app/condo/index')
+const { keystone } = index
+
+const taskRunTimeoutMs = 30000
```

Comment:
we already have a global timeout! you don't need to add any others timeouts. Use `--testTimeout=60000` options locally

Reply:
please remove this